### PR TITLE
chore(test): update connector integration tests

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Update connector ITs:
1. add test cases: send test email/SMS messages before setting passwordless connectors
2. chore: clarify the usage of `id` and `connectorId` in API util functions.
3. refactor: set the connector instances config upon creating.
4. refactor: add util function for clearing connector table.
5. rename: rename `mockStandardEmailConnector*` as `mockAlternativeEmailConnector*` to avoid confusing since passwordless connectors can never be "standard" per our design.

## Testing

UTs and ITs.

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/integration-tests/src/api/connector.ts [hunk 1]
```diff
@@ -2,13 +2,20 @@ import type { Connector, ConnectorResponse, CreateConnector } from '@logto/schem
 
 import { authedAdminApi } from './api.js';
 
+/**
+ * We are using `id` and `connectorId` here:
+ *
+ * - `id` is used to identify connectors from the database.
+ * - `connectorId` is used to identify connectors - more specifically, connector factories - in packages/connectors
+ * that contain metadata (considered connectors' FIXED properties) and code implementation (which determines how connectors work).
+ */
```
Comment (simeng-li) [inline packages/integration-tests/src/api/connector.ts:11]: Might need a better name? Is it widely used cross our connector code base that id stands for connector instances id in DB and connector stands for the factory?  A little bit confusing.  cc: @logto-io/eng

Comment (darcyYe) [reply packages/integration-tests/src/api/connector.ts:11]: In the IT, we can rename `connectorId` as `connectorFactoryId` and we can update the schema when we are going to boost to logto v2?

File: packages/integration-tests/src/__mocks__/connectors-mock.ts
Comment (gao-sun) [inline packages/integration-tests/src/__mocks__/connectors-mock.ts:1]: would you like to update "IT" to "integration test"?

File: packages/integration-tests/src/api/connector.ts (hunk 1 above)
Comment (darcyYe) [reply packages/integration-tests/src/api/connector.ts]: Added LOG-6104 to track this update.

File: packages/integration-tests/src/__mocks__/connectors-mock.ts
Comment (darcyYe) [reply packages/integration-tests/src/__mocks__/connectors-mock.ts:1]: Updated. Thanks for reminding me ❤️

File: packages/integration-tests/src/tests/api/connector.test.ts [hunk 2]
```diff
@@ -174,14 +183,20 @@ test('override metadata for non-standard social connector', async () => {
...
-  const connectors = await listConnectors();
-  await Promise.all(
-    connectors.map(async ({ id }) => {
-      await deleteConnectorById(id);
-    })
-  );
-  connectorIdMap.clear();
+  await cleanUpConnectorTable();
+
+  const phone = '8612345678901';
```
Comment (wangsijie) [inline packages/integration-tests/src/tests/api/connector.test.ts:188]: nit: use "generatePhone" and "generateEmail" utils

File: packages/integration-tests/src/tests/api/connector.test.ts [hunk 3]
```diff
@@ -174,14 +183,20 @@ test('override metadata for non-standard social connector', async () => {
...
+  const email = 'test@example.com';
+
+  // Can send test message without setting up passwordless connectors.
```
Comment (wangsijie) [inline packages/integration-tests/src/tests/api/connector.test.ts:191]: nit: a bit confusing here, how about: can send test message without enabling connector (send with config)

**Review by wangsijie — APPROVED** [review]

**Pull request merged**